app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -23,6 +36,75 @@
 
 public class BridgeUtils {
     
+    private static final Logger logger = LoggerFactory.getLogger(BridgeUtils.class);
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    /**
+     * Used for the Stormpath REST API. This should go away when they update their SDK.
+     * @param client
+     * @param url
+     * @param node
+     * @return
+     * @throws Exception
+     */
+    public static ObjectNode getJSON(CloseableHttpClient client, String url) throws Exception {
+        HttpGet get = new HttpGet(url);
+        get.setHeader("Accept", "application/json");
+        try(CloseableHttpResponse response = client.execute(get)) {
+            ObjectNode object = (ObjectNode)MAPPER.readTree(EntityUtils.toString(response.getEntity(), "UTF-8"));
+            if (logger.isDebugEnabled()) {
+                logger.debug("GET: " + url + "\n   request: <EMPTY>\n   response: "+ MAPPER.writeValueAsString(object));
+            }
```

Comment:
We should move the debug to before we parse JSON, so that way if the JSON parsing fails, we still print out the string we tried to parse.